strategy-cache-and-update/README.md

```diff
@@ -0,0 +1,22 @@
+# Cache and update
+The recipe provides a service worker answering from cache to deliver fast
```

Comment:
"... a service worker responding with data from the cache to deliver fast responses, and also updating the cache with fresh data from the network"